internal/controller/ols_app_server_deployment.go

```diff
@@ -54,6 +49,9 @@ func (r *OLSConfigReconciler) generateOLSDeployment(cr *olsv1alpha1.OLSConfig) (
 		secretMounts[provider.CredentialsSecretRef.Name] = credentialMountPath
 	}
 
+	redisSecretName := cr.Spec.OLSConfig.ConversationCache.Redis.CredentialsSecretRef.Name
+	redisCredentialsMountPath := path.Join(CredentialsMountRoot, redisSecretName)
```

Comment:
We refer these in both ols_app_server Pod Spec and Redis Pod Spec ,its good to move building of these common SecretName,MountPath, RedisCAVolume to a single common file. Will reduce our code foot print.

Reply:
Modularised the common parts of it. Thank you.